ui/components/multichain/connected-site-menu/connected-site-menu.js

```diff
@@ -41,7 +41,7 @@ export const ConnectedSiteMenu = ({ className, disabled, onClick, status }) => {
   const iconElement = currentTabHasNoAccounts ? (
     <Icon
       name={IconName.Global}
-      size={IconSize.Sm}
+      size={IconSize.Lg}
```

Comment:
Update globe icon to match updated hamburger menu.
<img width="420" height="87" alt="Screenshot 2025-08-22 at 21 00 30" src="https://github.com/user-attachments/assets/979d0e8e-ea91-4ad4-9838-c2bc19cff958" />